census/estimator/trainer/task.py

```diff
@@ -11,165 +11,173 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Runs the training of the Wide & Deep model based on hyperparameter
+values received as input parameters.
+"""
 
 import argparse
 import json
 import os
 
 import tensorflow as tf
 
+import trainer.input as input_module
 import trainer.model as model
 
 
 def _get_session_config_from_env_var():
-  """Returns a tf.ConfigProto instance that has appropriate device_filters set.
+    """Returns a tf.ConfigProto instance that has appropriate device_filters
+    set."""
 
-  """
+    tf_config = json.loads(os.environ.get('TF_CONFIG', '{}'))
 
-  tf_config = json.loads(os.environ.get('TF_CONFIG', '{}'))
-
-  if (tf_config and 'task' in tf_config and 'type' in tf_config['task'] and
-      'index' in tf_config['task']):
-    # Master should only communicate with itself and ps
-    if tf_config['task']['type'] == 'master':
-      return tf.ConfigProto(device_filters=['/job:ps', '/job:master'])
-    # Worker should only communicate with itself and ps
-    elif tf_config['task']['type'] == 'worker':
-      return tf.ConfigProto(device_filters=[
-          '/job:ps',
-          '/job:worker/task:%d' % tf_config['task']['index']
-      ])
-  return None
+    if (tf_config and 'task' in tf_config and 'type' in tf_config['task'] and
+            'index' in tf_config['task']):
+        # Master should only communicate with itself and ps
+        if tf_config['task']['type'] == 'master':
+            return tf.ConfigProto(device_filters=['/job:ps', '/job:master'])
+        # Worker should only communicate with itself and ps
+        elif tf_config['task']['type'] == 'worker':
+            return tf.ConfigProto(device_filters=[
+                '/job:ps',
+                '/job:worker/task:%d' % tf_config['task']['index']
+            ])
+    return None
 
 
 def train_and_evaluate(args):
-  """Run the training and evaluate using the high level API."""
```

Comment:
I am not sure if Github's guidance are 2 or 4 spaces. Will let cloudml-sample team confirm that one.

Reply:
Got it! I'll change if necessary.